src/middlewared/middlewared/etc_files/local/snmpd.conf

```diff
@@ -0,0 +1,34 @@
+<%
+    import subprocess
+
+    hw_machine = subprocess.check_output("/sbin/sysctl -n hw.machine", shell=True, encoding="utf-8").strip()
+    hw_model = subprocess.check_output("/sbin/sysctl -n hw.model", shell=True, encoding="utf-8").strip()
+    hw_clockrate = subprocess.check_output("/sbin/sysctl -n hw.clockrate", shell=True, encoding="utf-8").strip()
+    kern_ostype = subprocess.check_output("/sbin/sysctl -n kern.ostype", shell=True, encoding="utf-8").strip()
+    kern_osrelease = subprocess.check_output("/sbin/sysctl -n kern.osrelease", shell=True, encoding="utf-8").strip()
+    kern_osrevision = subprocess.check_output("/sbin/sysctl -n kern.osrevision", shell=True, encoding="utf-8").strip()
+    freenas_version = subprocess.check_output("cat /etc/version", shell=True, encoding="utf-8").strip()
+
+    config = middleware.call_sync("snmp.config")
+%>
+agentAddress udp:161,udp6:161,unix:/var/run/snmpd.sock
+sysLocation ${config["location"] or "unknown"}
+sysContact ${config["contact"] or "unknown@localhost"}
+sysDescr ${freenas_version}. Hardware: ${hw_machine} ${hw_model} running at ${hw_clockrate}. Software: ${kern_ostype} ${kern_osrelease} (revision ${kern_osrevision})
+
+master agentx
+
+% if config["v3"]:
+    % if config["v3_username"] and config["v3_password"]:
+createUser ${config["v3_username"]} ${config["v3_authtype"]} "${config["v3_password"]}" \
```

Comment:
We have a trailing `\`

Reply:
This is intentional so `${config["v3_privproto"]} "${config["v3_privpassphrase"]}"` stays on the same line

Reply:
Oh yup, missed that. Thank you for pointing it out